src/integrations/stories/ActionSidebar.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { type Meta, type StoryObj } from '@storybook/react';
```

Comment:
What's the top level `integrations` folder going to be used for? Just the stories?

Reply:
Yeah, for now this is the spot where I think the actual integrations will end up that people write. Eventually it'd be good to maybe even have an own repository for that but for now I think this is the easiest way. Maybe this will even change when I refactor the ActionSidebar and/or write the first integration.